Cauldron/Assets/Scripts/VisitorManager.cs

```diff
@@ -92,16 +94,20 @@ public async void Enter(Encounter card)
                     ignoreSavedAttempts = true;
                     break;
             }
-            VisitorEntering?.Invoke();
-            visitorTimer.ResetTimer(villager.patience);
-            if (attemptsLeft != villager.patience)
+
+            if (VisitorEntering == null)
```

Comment:
This is bad))
Let's make an int patience override variable and move all the logic here. GameMode will just set the override.
That would be a lot nicer then some random delegate with some custom logic switch))